app/Http/Controllers/Controller.php

```diff
@@ -10,4 +10,14 @@
 class Controller extends BaseController
 {
     use AuthorizesRequests, DispatchesJobs, ValidatesRequests;
+
+    $dates =[];
+
+    public function setDate(){
+      $date = filter_input(INPUT_POST, 'test_date');
```

Comment:
Don't worry about filter input and stuff like this. Its a prototype so "Just the Facts Ma'am"